src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/FlowCacheBoltCarrier.java

```diff
@@ -29,5 +29,5 @@ public interface FlowCacheBoltCarrier {
 
     void emitCheckFlowLatencyRequest(String flowId, FlowDirection direction, Duration latency);
 
-    public void emitLatencyStats(String flowId, FlowDirection direction, Duration latency);
+    void emitLatencyStats(String flowId, FlowDirection direction, Duration latency);
```

Comment:
This method stores metrics in OpenTsdb. FlowCacheBolt creates metric object (`emitLatencyStats`). THis metric has tag `flowid`. In case of HAFlow this field will contain id of the sub flow. It's okay, but not enough. For ha sub flow it should contain tag `ha_flow_id`. We hast have ability to find it by haflow